### PR TITLE
[Improvement][E2E] add e2e javatask case

### PR DESCRIPTION
**Pull request by jackyyyyyssss**
## Purpose of the pull request
add e2e java test  #15478 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by SbloodyS — CHANGES_REQUESTED** [review]
It's better to split bugfix and feature into two PR. @jackyyyyyssss

**Comment by jackyyyyyssss** [comment]
> It's better to split bugfix and feature into two PR. @jackyyyyyssss

ok .I will submit another PR to fix the bug in the Javatask classpath

**Review by songjianet — APPROVED** [review]

File: dolphinscheduler-standalone-server/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-FROM eclipse-temurin:8-jre
+FROM eclipse-temurin:8-jdk
```
Comment (SbloodyS) [inline dolphinscheduler-standalone-server/src/main/docker/Dockerfile:18]: Why did you change this?

Comment (jackyyyyyssss) [reply dolphinscheduler-standalone-server/src/main/docker/Dockerfile:18]: Because the JRE image does not have the installation of the JavaC command, the Javatask needs to be compiled before execution
image

Comment (SbloodyS) [reply dolphinscheduler-standalone-server/src/main/docker/Dockerfile:18]: I got it. Thanks.

**Review by SbloodyS — APPROVED** [review]
+1
